projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -58,6 +63,10 @@ AdminPage.defaultProps = {
 	showHeader: true,
 	showFooter: true,
 	showBackground: true,
+	logo: {
+		height: 32,
+		showText: true,
+	},
```

Comment:
These are already the default ones at `JetpackLogo`: https://github.com/Automattic/jetpack/blob/master/projects/js-packages/components/components/jetpack-logo/index.jsx#L22-L27 ... You should make `logo` an `empty` object only, not?

Reply:
yes, I know. I left there just to show some of the Logo properties. making logo an empty object is ok too

Reply:
I prefer to keep these default values there, Renato. it shows how to use them. Do you think it's acceptable?

Reply:
Yes, no problem. Was just thinking in the case we update JetpackLogo default values, but don't think it will be a big problem